build/config.props

```diff
@@ -90,11 +90,6 @@
     <PackageIcon>icon.png</PackageIcon>
   </PropertyGroup>
 
-  <PropertyGroup>
-    <NUGET_PFX_PATH>$(MSBuildThisFileDirectory)..\keys\NuGetKey.snk</NUGET_PFX_PATH>
-    <MS_PFX_PATH>$(MSBuildThisFileDirectory)..\keys\35MSSharedLib1024.snk</MS_PFX_PATH>
```

Comment:
build.ps1 override these when `-SkipDelaySign` was passed as an argument. It should be updated.

It possibly also makes this PR redundant, since there's already a different way to achieve the same outcome?

Reply:
Thanks for pointing that out. I've updated the script to set `SkipSigning=true` to just disable all strong name signing.  This pull request will make it easier for our CI on all platforms to disable signing.  I'm thinking in the near future to instead use `SignType` values of `None` (default), `Test`, and `Real`.  This would be more in line with MicroBuild.  But from a developer's perspective, we just build and run tests.  I need to find a way to make it so we don't have to do anything special to debug our VSIX (if possible) and have the default signing to None.  Otherwise I'll default signing to "Test" which will delay sign things and require strong name signing validation exclusions.  But for Linux and Mac CI, we'll just build and run tests as unsigned.